src/prefect/agent/fargate/agent.py

```diff
@@ -14,6 +14,7 @@ class FargateAgent(Agent):
     Fargate Agent can be found at https://docs.prefect.io/cloud/agent/fargate.html
 
     Args:
+
```

Comment:
```suggestion
```

Reply:
I think this extra space is what is causing the documentation formatting error :)